src/aliases/api/handlers/handlers.go

```diff
@@ -0,0 +1,299 @@
+// @Success 200 {object} types.CreateAliasResponse "Alias data"
+// @Failure 400 {object} ErrorResponse "Invalid request format"
+// @Failure 500 {object} ErrorResponse "Internal server error"
+// @Router /aliases/{registry_name}/aliases/{alias_key} [post]
+func (h *AliasHandler) createAlias(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	// should always exists in this subrouter
+	regName := vars["registry_name"]
+	key := vars["alias_key"]
+
+	err := validatePathVars(regName, key)
+	if err != nil {
+		infrahttp.WriteHTTPErrorResponse(w, errors.InvalidFormatError(err.Error()))
+		return
+	}
+
+	var aliasReq types.AliasRequest
+	err = jsonutils.UnmarshalBody(r.Body, &aliasReq)
+	if err != nil {
+		infrahttp.WriteHTTPErrorResponse(w, errors.InvalidFormatError(err.Error()))
+		return
+	}
+
+	eAlias := types.FormatAlias(types.RegistryName(regName), key, aliasReq.Value)
+	alias, err := h.alias.CreateAlias(r.Context(), eAlias.RegistryName, eAlias)
+	if err != nil {
+		infrahttp.WriteHTTPErrorResponse(w, err)
+		return
+	}
+
+	resp := types.AliasResponse{
+		Value: types.AliasValue(alias.Value),
+	}
+	err = jsonWrite(w, resp)
+	if err != nil {
+		infrahttp.WriteHTTPErrorResponse(w, err)
+		return
+	}
+}
+
+// @Summary Get an alias
+// @Description Get an alias of a key from a dedicated alias registry
+// @Tags alias
+// @Produce json
+// @Param registry_name path string true "registry identifier"
+// @Param alias_key path string true "alias identifier"
+// @Success 200 {object} types.GetAliasResponse "Alias data"
+// @Failure 400 {object} ErrorResponse "Invalid request format"
+// @Failure 500 {object} ErrorResponse "Internal server error"
+// @Router /aliases/{registry_name}/aliases/{alias_key} [get]
+func (h *AliasHandler) getAlias(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	// should always exists in this subrouter
+	regName := vars["registry_name"]
+	key := vars["alias_key"]
+
+	err := validatePathVars(regName, key)
+	if err != nil {
+		infrahttp.WriteHTTPErrorResponse(w, errors.InvalidFormatError(err.Error()))
+		return
+	}
+
+	alias, err := h.alias.GetAlias(r.Context(), aliasent.RegistryName(regName), aliasent.AliasKey(key))
+	if err != nil {
+		infrahttp.WriteHTTPErrorResponse(w, err)
+		return
+	}
+
+	err = jsonWrite(w, types.AliasResponse{
+		Value: types.AliasValue(alias.Value),
+	})
+	if err != nil {
+		infrahttp.WriteHTTPErrorResponse(w, err)
+		return
+	}
+}
+
+// updateAlias updates an alias value.
+// @Summary Update an alias
+// @Description Update an alias of a key from a dedicated alias registry
+// @Tags alias
+// @Accept json
+// @Produce json
+// @Param registry_name path string true "registry identifier"
+// @Param alias_key path string true "alias identifier"
+// @Param request body types.UpdateAliasRequest true "Update Alias Request"
+// @Success 200 {object} types.UpdateAliasResponse "Alias data"
+// @Failure 400 {object} ErrorResponse "Invalid request format"
```

Comment:
I believe a 404 is also a possible error

Reply:
Yes!